docs/core/install/linux-fedora.md

```diff
@@ -104,7 +92,11 @@ This section provides information on common errors you may get while using the p
 
 ### Unable to find package
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+For more information on installing .NET without a package manager, see one of the following articles:
+
+- [Install the .NET SDK or the .NET Runtime with Snap.](../linux-snap.md)
+- [Install the .NET SDK or the .NET Runtime with a script.](../linux-scripted-manual.md#scripted-install)
+- [Install the .NET SDK or the .NET Runtime manually.](../linux-scripted-manual.md#manual-install)
 
 ### Failed to fetch
 
```

Comment:
Github wont let me comment there but line 108 talks about installing tab completion. The Fedora packages set that up automatically, for `bash`. So it's not required for the user to do that manually; it should work out of the box.

`zsh` completion is blocked on something like https://github.com/dotnet/sdk/pull/14239.

Reply:
Good point. Perhaps I should add something there about first testing if tab completion is working